extras/rya.streams/client/src/main/java/org/apache/rya/streams/client/command/ListQueriesCommand.java

```diff
@@ -107,12 +112,11 @@ private String formatQueries(final Set<StreamsQuery> queries) {
         sb.append("Queries in Rya Streams:\n");
         sb.append("---------------------------------------------------------\n");
         queries.forEach(query -> {
-            sb.append("ID: ");
-            sb.append(query.getQueryId());
-            sb.append("\t\t");
-            sb.append("Query: ");
-            sb.append(query.getSparql());
-            sb.append("\n");
+            sb.append("ID: ").append(query.getQueryId())
+                .append("    ")
+                .append("Is Active: ").append(query.isActive())
+                .append( query.isActive() ? "     " : "    " )
```

Comment:
Maybe change to:

<code>
.append("Is Active: ")

.append(StringUtils.rightPad(query.isActive(), 9))
</code>

Reply:
Done.